src/test/resources/data/AutomatedSessionRemindersTest.json

```diff
@@ -22,7 +22,7 @@
       "googleId": "AutSessRem.instructor",
       "courseId": "AutSessRem.course",
       "name": "Test Ins for Aut Sessions Reminder",
-      "email": "${support.email}",
+      "email": "${test.instructor}@gmail.com",
```

Comment:
I think now I recall why the `support.email` is used here. It's because during live testing emails are actually sent out which end up in my email box. So when I do a live testing, I expect to see a certain set of emails in my email box. If I don't see them, I know there is a breakdown in our email sending. So the original one was correct, but it was not documented properly. This manual check while live testing is important because email sending is skipped in dev server testing.